internalsdk/auth/auth.go

```diff
@@ -50,25 +46,22 @@ type AuthClient interface {
 }
 
 // NewClient creates a new instance of AuthClient
-func NewClient(baseURL string, opts *webclient.Opts) AuthClient {
-	httpClient := opts.HttpClient
-	if httpClient == nil {
-		// The default http.RoundTripper is ChainedNonPersistent which proxies requests through chained servers
-		// and does not use keep alive connections. Since no root CA is specified, we do not need to check for an error.
-		rt, _ := proxied.ChainedNonPersistent("")
-		httpClient = pro.NewHTTPClient(rt, opts)
-	}
-
-	wc := webclient.NewRESTClient(defaultwebclient.SendToURL(httpClient, baseURL,
-		func(client *resty.Client, req *http.Request) error {
-			prepareUserRequest(req, opts.UserConfig())
+func NewClient(baseURL string, userConfig func() common.UserConfig) AuthClient {
+	// The default http.RoundTripper is ChainedNonPersistent which proxies requests through chained servers
+	// and does not use keep alive connections. Since no root CA is specified, we do not need to check for an error.
+	// Temporarily disable use of ChainedNonPersistent
+	// rt, _ := proxied.ChainedNonPersistent("")
```

Comment:
We can continue using a proxied.ParallelForIdempotent http.RoundTripper for the auth API. I tested it works fine, but I'm leaving the comment here in case we need to re-enable it.